{{cookiecutter.repo_name}}/circle.yml

```diff
@@ -9,3 +20,5 @@ database:
 test:
   override:
      - tox
+  post:
```

Comment:
@adamhadani have you tried moving the ecr push to the tests section of the circleci file anywhere? I remember you wanted to try it rather than doing it in the `dependencies` step.